src/crimson/osd/watch.h

```diff
@@ -53,4 +69,53 @@ class Watch : public std::enable_shared_from_this<Watch> {
 
 using WatchRef = Watch::Ref;
 
+class Notify {
+  std::set<WatchRef> watchers;
```

Comment:
i'd suggest move the private members to the end of function declaration. see https://google.github.io/styleguide/cppguide.html#Declaration_Order

Reply:
ACK, will move.

Reply:
I propose to do the clean-up in a separate commit.